src/mame/mame.lst

```diff
@@ -45469,6 +45469,7 @@ gjrstar3                        // Genius Junior Redstar 3 (Germany)
 gln                             // Genius Leader Notebook
 pitagor                         // Pitagorin
 gls                             // Genius Leader Select
+pcompelr                        // Precomputer Electronik (Russia)
 pitagjr                         // Pitagorin Junior
 scientus                        // Scientus (France, from C.Q.F.D)
```

Comment:
Please get these back in alphabetical order.

Reply:
Fixed! (it wasn't ordered before, I'd say...)